bin/before_script.sh

```diff
@@ -2,45 +2,61 @@
 
 # before_script
 export PATH="$HOME/.composer/vendor/bin:$PATH"
-export REPOSITORY_NAME=$(find $TRAVIS_BUILD_URL -mindepth 1 -maxdepth 1 -name "*.info" -type f -printf '%f\n' | cut -f1 -d".")
+export REPOSITORY_NAME=$(basename $TRAVIS_BUILD_DIR)
+
 # download linky_clicky and copy over related tests and required files
 if [ ! -z "$CLICKY_BRANCH" ]; then CLICKY_BRANCH="-b $CLICKY_BRANCH"; fi
+echo "git clone --depth 1 $CLICKY_BRANCH https://github.com/SU-SWS/linky_clicky.git $HOME/linky_clicky"
```

Comment:
Yes, for easy debugging.  Even though Travis prints custom variables before starting a build, I found myself second guessing whether the right branches were really being downloaded.